pkgs/development/libraries/aws-sdk-cpp/default.nix

```diff
@@ -25,15 +25,21 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ cmake curl ];
 
   buildInputs = [
-    curl openssl s2n-tls zlib
-    aws-c-common aws-c-event-stream aws-checksums
+    curl openssl zlib
   ] ++ lib.optionals (stdenv.isDarwin &&
                         ((builtins.elem "text-to-speech" apis) ||
                          (builtins.elem "*" apis)))
          [ CoreAudio AudioToolbox ];
 
   # propagation is needed for Security.framework to be available when linking
-  propagatedBuildInputs = [ aws-c-cal aws-c-io ];
+  propagatedBuildInputs = [
+    aws-c-cal
+    aws-c-event-stream
+    aws-c-io
+    aws-c-common
+    aws-checksums
+    s2n-tls
```

Comment:
Can we clean those up from depending packages?

Reply:
It looks like the only thing that depends on `aws-sdk-cpp` is nix itself, optionally.

Reply:
So I guess the answer to your question is no, since there aren't any dependents in nixpkgs as far as I can see.